CHANGELOG.md

```diff
@@ -1,5 +1,62 @@
 # Change Log
 
+## [0.8.0](https://github.com/angelozerr/lsp4xml/milestone/9?closed=1) (July 23, 2019)
+
+### Enhancements
+
+ * Find definition for start/end tag element. See[#535](https://github.com/angelozerr/lsp4xml/issues/535)
+ * Improve Source information from completion. See[#529](https://github.com/angelozerr/lsp4xml/issues/529)
+ * `XSD CodeLens` references should be clickable. See[#490](https://github.com/angelozerr/lsp4xml/issues/490)
+ * Add support for `textDocument/documentHighlight` for XML Schema types. See[#470](https://github.com/angelozerr/lsp4xml/issues/470)
+ * Add support for `textDocument/completion` for xs:element/@name / xs:extension/@base. See[#451](https://github.com/angelozerr/lsp4xml/issues/451)
+ * Support selective enablement of outline per file. See[#427](https://github.com/angelozerr/lsp4xml/issues/427)
+ * `.ent` and `.mod` files should be parsed as DTDs. See[#380](https://github.com/angelozerr/lsp4xml/issues/380)
+ * Add support for `textDocument/typeDefinition` from XML to XMLSchema/DTD . See[#371](https://github.com/angelozerr/lsp4xml/issues/371)
+ * Show possible elements for completion based on XML Schema. See[#347](https://github.com/angelozerr/lsp4xml/issues/347)
+ * Add support for `textDocument/definition` for XML Schema. See[#148](https://github.com/angelozerr/lsp4xml/issues/148)
+ * Add support for `textDocument/references` for XML Schema types. See[#58](https://github.com/angelozerr/lsp4xml/issues/58)
+ * Add support for `textDocument/codelens` for XML Schema types. See[#55](https://github.com/angelozerr/lsp4xml/issues/55)
+ * Add completion for `comment` and `#region`. See[#54](https://github.com/angelozerr/lsp4xml/issues/54)
+ * CData block auto-complete. See[#168](https://github.com/redhat-developer/vscode-xml/issues/168)
+ * Documentation in completion has monospace font. See[#32](https://github.com/redhat-developer/vscode-xml/issues/32)
```

Comment:
I would suggest: "Add markdown support for hover text and completion descriptions"

Reply:
@xorye updated